ui/src/main/java/edu/wpi/grip/ui/preview/PreviewsController.java

```diff
@@ -26,20 +29,20 @@
 @Singleton
 public class PreviewsController {
 
-    @FXML private HBox previewBox;
+    @FXML
```

Comment:
Yes, almost all of them seem to be random unnecessary refactoring to your preferred level of abstraction and coding style.